about.html

```diff
@@ -0,0 +1,108 @@
+                <li><a href="about.html" class="navLink">About</a></li>
+                <li><a href="" class="navLink">Program</a></li>
+                <li><a href="" class="navLink">Join</a></li>
+                <li><a href="" class="navLink">Sponser</a></li>
+                <li><a href="" class="navLink">News</a></li>
+                <li><button>CC Campiaign</button></li>
+              </ul>
+            </div>
+      <h3 class="heading-main">"Welcome to Faya Beauty Salon"</h3>
+      <h1>COME HERE TO LOOK <br> GLOW and SHINE</h1>
+      <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. <br> Eius cumque, illo at ad veniam, delectus facilis qui quod, excepturi id iusto doloremque eveniet molestias eos! Reiciendis amet deleniti distinctio cum?</p>
+      <p>please contact us per Email for any further questions <br> about faya women beauty salon <br>
+      fayasalon@gmail.com</p>
+    </div>
+    <div class="about-logo-section">
+      <h1>Faya womens Beauty Salon Logo</h1>
+      <hr class="about-section-separator">
+      <p>The logo of faya womens beauty salon was decided <br>by the national trade agency</p>
+      <img src="image/logo.png" alt="">
+      <hr class="about-logo-section-separator">
+      <h1>See the past faya women beauty salon works</h1>
+      <hr class="about-section-separator">
+      <p class="prvious-work">Take a look at the last two faya womens beauty salon works with the most profitable companies in Addis Ababa </p>
+      <div class="previous-work-list">
+        <div class="previous-work">
+          <div class="cover-image">
+            <div class="cover-text">
+              <h3 class="heading-pre-work">2021</h3>
+              <h4>Faya beauty salon Bridal work</h4>
+            </div>
+          </div>
+        </div>
+        <div class="previous-work">
+          <div class="cover cover-image">
+            <div class="cover-text">
+              <h3 class="heading-pre-work">2022</h3>
+              <h4>Faya beauty salon Bridal work</h4>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="about-partner">
+        <h3>Partner</h3>
+        <hr class="about-partner-separator">
+        <ul class="partners-list">
+          <li><a href="" class="partners-list-log"><img src="image/afri-herbal.jpg" alt=""></a></li>
+          <li><a href="" class="partners-list-log"><img src="image/sol-image.png" alt=""></a></li>
+          <li><a href="" class="partners-list-log"><img src="image/barber-chair.jpg" alt=""></a></li>
+          <li><a href="" class="partners-list-log"><img src="image/ethi-herbal.jpg" alt=""></a></li>
+          <li><a href="" class="partners-list-log"><img src="image/awash.png" alt=""></a></li>
+
+        </ul>
+      </div>
+      <div class="footer-logo-section">
+        <img src="image/logo.png" alt="">
+        <p class="footer-logo-section-para">Faya Womens Beauty Salon Some Rights Reserved</p>
+      </div>
```

Comment:
- [ ] Please kindly fix the design of this section to meet the required project design [required project design ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Your design**
![screen](https://user-images.githubusercontent.com/55596970/170740960-fa411865-0371-47e1-82ca-0d7b9a2a8858.png)